src/features/User/Ui/NavBar/NavBar.tsx

```diff
@@ -1,45 +1,140 @@
-            // onClick={handleMenu}
-            // color="inherit"
-          >
-            <Avatar alt="Remy Sharp" src={avatar }/>
-          </IconButton>
-        </Toolbar>
+      <AppBar
+        elevation={0}
+        color="transparent"
+        // sx={{ml:"250px"}}
+        position="static"
+      >
+        <Container 
+        component="navbar"
+        maxWidth="l"
+         style={{ justifyContent: "end" }}
+         >
+          <Toolbar style={{ justifyContent: "space-between" }}>
+
+            <Typography
+              variant="h6"
+              noWrap
+              component="a"
+              href="#app-bar-with-responsive-menu"
+              sx={{
+                mr: 2,
+                display: { xs: "none", md: "flex" },
+                fontFamily: "monospace",
+                fontWeight: 700,
+                letterSpacing: ".3rem",
+                color: "inherit",
+                textDecoration: "none",
+                fontSize: "1.8rem" 
+              }}
+            >
+              <Box
+              component="span"
+              sx={{ color: "#007BFF", fontWeight: 'bold' }}
+            >
+              Stay
+            </Box>
+            <Box component="span" sx={{ color: "black", fontWeight: 'bold' }}>
+              cation.
+            </Box>
+            </Typography>
+            <Box sx={{ display: "flex" }}>
+              <Box
+                sx={{
+                  flexGrow: 0,
+                  display: { xs: 'none', md: 'flex' },
+                  // display: "flex",
+                }}
+              >
+                <MenuItem>Home</MenuItem>
+                <MenuItem>Rooms</MenuItem>
+                <MenuItem>Favorites</MenuItem>
+              </Box>
+              <Box>
+                <Typography
+                  sx={{
+                    "&:hover": {
+                      color: "red",
+                      // fontSize: "24px"
+                    },
+                  }}
+                  // color="inherit"
+                >
+                  {/* user */}
+                  {/* {userData?.userName || "user"}*/}
+                </Typography>
+                <Button
+                  id="fade-button"
+                  size="large"
+                  aria-controls={open ? "fade-menu" : undefined}
+                  aria-haspopup="true"
+                  aria-expanded={open ? "true" : undefined}
+                  onClick={handleClick}
+                >
+                  <Avatar alt="Remy Sharp" src={avatar} />
+                </Button>
+                <Menu
+                  id="fade-menu"
+                  MenuListProps={{
+                    "aria-labelledby": "fade-button",
+                  }}
+                  anchorEl={anchorEl}
+                  open={open}
+                  onClose={handleClose}
+                  TransitionComponent={Fade}
+                >
+                  <MenuItem onClick={handleClose}>Profile</MenuItem>
```

Comment:
There are additional navigations and a login button added to the Figma design, you might need to consider it in the design.

![image](https://github.com/SAMIYAghb/hotel-system/assets/64327685/d14d6727-4abe-4430-8234-97a3ac47f02a)